containers/kubernetes/how-to/access-kubernetes-dashboard.mdx

```diff
@@ -0,0 +1,44 @@
+---
+meta:
+  title: How to access the dashboard of a Kubernetes Kapsule cluster
+  description: This page explains how to access the dashboard of your Kubernetes Kapsule cluster
+content:
+  h1: How to access the dashboard of a Kubernetes Kapsule cluster
+  paragraph: This page explains how to access the dashboard of your Kubernetes Kapsule cluster
+tags: kubernetes kubernetes-kapsule kapsule cluster 
+dates:
+  validation: 2023-02-07
+  posted: 2020-10-14
+categories:
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+## How to access the Kubernetes dashboard
+
+1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
```

Comment:
Maybe confusing that we talk about a Kubernetes dashboard here, but it's another type of dashboard!